webplugin/js/app/kommunicate.js

```diff
@@ -790,29 +790,16 @@ $applozic.extend(true, Kommunicate, {
         return true;
     },
     hideMessage: function (element) {
-        if (!element) {
-            return;
-        }
-        var parentEle = element.parentElement;
-        while (!parentEle.classList.contains('mck-msg-left')) {
-            parentEle = parentEle.parentElement;
-        }
-        parentEle.classList.add('n-vis');
-
-        var quickReplyCtaPrevSibling = parentEle.previousElementSibling;
+        //all cta expect link button will hide if hidePostCta is enable
+        if (!element && !element[0]) return;
```

Comment:
should it be or check?

Reply:
no, because if element is undefined, null then element[0] is also undefined, null resp., so or check will not work here